core/src/test/resources/sql/functions.iq

```diff
@@ -938,5 +938,19 @@ table(AUX.TBLFUN_IDENTITY(3)) as t3(v);
 
 !ok
 
+# SUBSTRING
```

Comment:
I suggest including a test for the 2-arg form too. Like:

```
select SUBSTRING('text' FROM NULL)
```

It should return null.

Reply:
done